bazel/repositories.bzl

```diff
@@ -315,6 +317,16 @@ def _net_zlib():
         actual = "@envoy//bazel/foreign_cc:zlib",
     )
 
+def _com_google_cel_cpp():
+    location = REPOSITORY_LOCATIONS["com_google_cel_cpp"]
+    http_archive(
+        name = "com_google_cel_cpp",
+        # TODO(kyessenov): requires C++17 partial template specialization available in clang-8 but not gcc
```

Comment:
I'm not keen to keep this in Envoy repo, as this seems a general open-source issue of cel-cpp. I understand that this is not an issue in google3 but as a open-sourced project it is general enough to upstreamed. I'm fine if other (possibly Google) maintainer is up for this. @htuch?

Reply:
It's be ideal to use `absl::bind_front` but absl keeps delaying the release of important libraries. I'm happy to save this workaround in google3 once bind_front comes out.

Reply:
Yeah, can we upstream this patch?

Reply:
I'm not happy about the patch since it's almost 2x slower than the fast path. Can we postpone it till we get bind_front in absl (any day now TM)? I'm going to continue to maintain the integration, and it should be easier to reconcile the differences.